rpmlb/cli.py

```diff
@@ -1,24 +1,24 @@
 """CLI interface for the package"""
 
+import logging
 import os
 
 import click
 
-from . import LOG, configure_logging
+from . import logging as rpmlb_logging
 from .builder.base import BaseBuilder
 from .downloader.base import BaseDownloader
 from .recipe import Recipe
 from .work import Work
 
+LOG = None
+
 
 @click.command()
 # General options
 @click.option(
     '--verbose', '-v', is_flag=True, default=False,
     help='Turn on verbose logging.',
-    # Enable logging as early as possible
-    is_eager=True, expose_value=False,
-    callback=lambda ctx, param, verbose: configure_logging(verbose),
```

Comment:
OK, could you send your PR to my branch: https://github.com/junaruga/rpm-list-builder/tree/feature/nit-picking-make-init.py-empty ?
Because I can not imagine your preferred implementation in this time.
I would accept the PR.